doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -325,6 +325,15 @@ only as a storage format. In calculations they'll be converted to ``Float32``:
     julia> 2*Float16(4.)
     8.0f0
 
+The underscore ``_`` can be used as digit separator:
+
+.. doctest::
+
+    julia> 1_000
+    1000
+
+    julia> 10_000.10_001
```

Comment:
Maybe in front, because looking at it again, it is really in the middle of the floating point section.